app/views/products/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<h2><%= @vendor.vendor_name %></h2>
+<h3> Add a Product </h3>
+
+<% vendor = Vendor.find(params[:id]) %>
```

Comment:
Thank you. I'm having a hard time figuring out what is passed through to where, and would welcome any resources that you found useful in deciphering this.